nvqm.h

```diff
@@ -1152,6 +1156,10 @@ static inline xvec2 xvec2_clamp(xvec2 a, xvec2 min, xvec2 max){
 	};
 }
 
+static inline xint xvec2_cross(xvec2 a, xvec2 b){
+	return a.v[0] * b.v[1] - a.v[1] * b.v[0];
```

Comment:
this should use `xint_mul` and `xint_sub`, so:

```
return xint_sub(xint_mul(a.v[0], b.v[1]), xint_mul(a.v[1], b.v[0]));
```

Reply:
oops